mmcls/models/backbones/vision_transformer.py

```diff
@@ -221,41 +83,123 @@ def norm1(self):
     def norm2(self):
         return getattr(self, self.norm2_name)
 
+    def init_weights(self):
+        super(TransformerEncoderLayer, self).init_weights()
+        for m in self.ffn.modules():
+            if isinstance(m, nn.Linear):
+                nn.init.xavier_normal_(m.weight)
+                nn.init.normal_(m.bias, std=1e-6)
+
+    def forward(self, x):
+        x = self.attn(self.norm1(x), identity=x)
+        x = self.ffn(self.norm2(x), identity=x)
+        return x
+
+
+class PatchEmbed_v2(BaseModule):
+    """Image to Patch Embedding V2.
+
+    We use a conv layer to implement PatchEmbed.
+
+    Args:
+        img_size (int | tuple): The size of input image. Default: 224
+        in_channels (int): The num of input channels. Default: 3
+        embed_dims (int): The dimensions of embedding. Default: 768
+        kernal_size (int | tuple): Size of the convolving kernel. Default: 16
+        stride (int | tuple): Stride of the convolution. Default: 16
+        padding (int | tuple): Padding added to all four sides of
+            the input. Default: 0
+        dilation (int | tuple): Spacing between kernel elements. Default: 1
+        norm_cfg (dict, optional): Config dict for normalization layer.
+        conv_cfg (dict, optional): The config dict for conv layers.
+            Default: None.
+        init_cfg (`mmcv.ConfigDict`, optional): The Config for initialization.
+            Default: None.
+    """
+
+    def __init__(self,
+                 img_size=224,
+                 in_channels=3,
+                 embed_dims=768,
+                 kernal_size=16,
+                 stride=16,
+                 padding=0,
+                 dilation=1,
+                 norm_cfg=None,
+                 conv_cfg=None,
+                 init_cfg=None):
+        super(PatchEmbed_v2, self).__init__(init_cfg)
+
+        self.img_size = to_2tuple(img_size)
+        self.embed_dims = embed_dims
+        kernal_size = to_2tuple(kernal_size)
+        stride = to_2tuple(stride)
+        padding = to_2tuple(padding)
+        dilation = to_2tuple(dilation)
+
+        patches_resolution = [
+            (self.img_size[i] + 2 * padding[i] - kernal_size[i]) // stride[i] +
+            1 for i in range(2)
+        ]
```

Comment:
It seems that dilation is missing (refer to [torch.nn.Unfold](https://pytorch.org/docs/stable/generated/torch.nn.Unfold.html?highlight=unfold#torch.nn.Unfold) or [torch.nn.Conv2d](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html?highlight=conv2d#torch.nn.Conv2d)).

Reply:
It seems that dilation is missing (refer to torch.nn.Unfold or torch.nn.Conv2d).